proposals/0013-expanded-comparison-sampling.md

```diff
@@ -0,0 +1,432 @@
+# Expanded Comparison Sampling
+
+* Proposal: [0013](0013-expanded-comparison-sampling.md)
+* Author(s): [Greg Roth](https://github.com/pow2clk),
+             [Jesse Natalie](https://github.com/jenatali)
+* Sponsor: [Greg Roth](https://github.com/pow2clk)
+* Status: **Under Consideration**
+* Planned Version: Shader Model 6.8, validator version 1.8
+* Issues: [30](https://github.com/microsoft/hlsl-specs/issues/30),
+          [81](https://github.com/microsoft/hlsl-specs/issues/81)
+
+## Introduction
+
+Sampler operations in HLSL perform either a comparison sampler or non-comparison
+ sampler operation.
+Comparison sample operations will perform the compare operation specified in the
+ sampler between the sampled value and a provided value and produce a 0 or 1
+ value indicating the failure or success of the comparison respectively.
+In linear filtered samples, multiple values will be blended to produce the
+ complete sample operation result.
+
+HLSL has sampling operations which sample from a texture according to the
+ settings in the provided sampler object.
+The most useful forms of sampling involve the calculation of a MIP level
+ to determine which level(s) of detail(LOD) of the sampled texture should be
+ used and how they should be weighted.
+
+Different non-comparison sampler operations calculate the MIP level in different
+ ways:
+
+* `Sample(SamplerState S, float<num> Location)`
```

Comment:
Maybe instead of `float<num>` which isn't valid syntax and is a bit odd, `vector<float, X>` which is valid-ish?